etl/steps/data/garden/faostat/2023-02-22/detected_anomalies.py

```diff
@@ -0,0 +1,358 @@
+"""Detected anomalies.
+
+This module contains a class for each data anomaly detected. The class has the following methods and properties:
+* description: A human-readable text that describes the anomaly.
+  NOTE: The description will be added to the dataset metadata description, and hence will be shown in grapher.
+* checks: A method that ensures the anomaly exists in the data.
+  This is useful to detect if an anomaly has been corrected after a data update.
+* fix: A method that removes the anomaly.
+* inspect: An optional method that plots (in the browser) a visualization that shows the anomaly.
+  It can be used before and after removing the anomalies.
+* handle_anomalies: A helper method that uses all the previous methods in the usual order.
```

Comment:
I'd probably have this text in the methods from the abstract class `DataAnomaly`, too, as one-line docstrings, for instance.